gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -104,7 +104,8 @@ public Color (Color color) {
 
 	/** Sets this color to the given color.
 	 * 
-	 * @param color the Color */
+	 * @param color the Color
+   * @return this color. */
```

Comment:
Ah, my nemesis, indentation! I would run `gradlew spotlessApply` and commit the changed file. It may initially look like thousands of changed files, but it is actually only going to commit this one.

Reply:
Should be fixed. Took me a bit to figure out that Java 21 is not supported...